src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -127,7 +139,10 @@ export const createHash = (data: BinaryLike | undefined, secret: string): string
 		.digest("hex")}`;
 };
 
-const getWebhookSecret = async (uuid?: string): Promise<{ webhookSecret: string, gitHubServerApp?: GitHubServerApp }> => {
+const getWebhookSecrets = async (uuid?: string): Promise<{ webhookSecrets: Array<string>, gitHubServerApp?: GitHubServerApp }> => {
+	// TODO: Remove after testing
+	const allowGhCloudWebhookSecrets = await booleanFlag(BooleanFlags.ALLOW_GH_CLOUD_WEBHOOKS_SECRETS);
```

Comment:
I think you missed the jiraHost here, otherwise you can't enable the flag just for your own instance.

---
Hmm, however, we might not be able to get the jiraHost ( or we shouldn't )? As it is pretty earlier in the stage, and even before signature verified, app doesn't pull DB to grab subscription ( and therefore the jiraHost).

Reply:
Thats a good point, we don't have jiraHost at this point.

Reply:
Added a FF, using the Github User Login as the key.